index.html

```diff
@@ -0,0 +1,159 @@
+1.- on Aside, remove margins
+2.- on Section, add float: left;   <- *main magic.
+3.- the top margin do it to the Main section.
+  - so it is nice and tidy you can do it like this:
+    margin: 58px 3% 2%;   <- top left&right bottom
+(actually because this works, you don't strictly need the float left on the aside but with it it floats better.)
```

Comment:
Please remove these comments from your code as they prevented me from viewing the live version of your project!

### NOTE TO SUBSEQUENT TSE: Please review the live project for potential changes to be made as I couldn't due to these comments. Thanks!